capstone/config/settings/settings_base.py

```diff
@@ -9,7 +9,7 @@
 SERVICES_DIR = os.path.join(os.path.dirname(BASE_DIR), 'services')
 
 
-ALLOWED_HOSTS = ['localhost', '127.0.0.1', '[::1]', '.test']
+ALLOWED_HOSTS = ['localhost', '127.0.0.1', '[::1]', '.test', '.web']
```

Comment:
Designed to allow http calls made in the format http://web:8000/ etc.,

Reply:
To work properly locally, CAP needs to be served from several [specific hosts](https://github.com/harvard-lil/capstone#hosts-setup-). A person can tweak `docker-compose.yml` so that, container to container, you can use those same hosts, instead of the `web` service name. You have to set aliases `network` like [this](https://github.com/harvard-lil/perma/blob/develop/docker-compose.yml#L60-L65) and `extra_hosts` (the equivalent of /etc/hosts) like [this](https://github.com/harvard-lil/perma/blob/develop/docker-compose.yml#L52-L55).

Then, you can, for instance, `curl http://cap.test:8000` from your laptop AND from inside any of the containers, e.g. `docker compose exec db curl http://cap.test:8000` (presuming curl is installed in the db container lol)